namer/marathon/src/main/scala/io/buoyant/namer/marathon/MarathonInitializer.scala

```diff
@@ -49,7 +52,18 @@ case class MarathonConfig(
     case None => 80
   }
   private[this] def getUriPrefix = uriPrefix.getOrElse("")
-  private[this] def getTtl = ttlMs.getOrElse(5000).millis
+
+  @JsonIgnore
+  private[this] def getJitteredNextTtl(ttlMs: Int, jitterMsConf: Int): Stream[Duration] = {
```

Comment:
I think you could simplify this to something like `Stream.continually(ttl + (scala.util.Random.nextDouble() * 2 - 1) * jitter)`